internal/handlers/pg/msg_createindexes.go

```diff
@@ -122,6 +122,13 @@ func (h *Handler) MsgCreateIndexes(ctx context.Context, msg *wire.OpMsg) (*wire.
 			"One of the specified indexes already exists with a different key",
 			document.Command(),
 		)
+	case errors.Is(err, pgdb.ErrUniqueViolation):
+		return nil, commonerrors.NewCommandErrorMsgWithArgument(
+			commonerrors.ErrDuplicateKeyInsert,
+			"Index build failed",
+			document.Command(),
+		)
```

Comment:
Could you add integration test for checking error message please. Example https://github.com/FerretDB/FerretDB/blob/main/integration/indexes_test.go#L176 :hugs:

Reply:
I can't reproduce it now. It was probably about creating an index for a not existing field, but we now have a separate [issue](https://github.com/FerretDB/FerretDB/issues/2830).

Reply:
Maybe this is related to below? I just used `Doubles` collection as it is from `task env-data` and it somehow does not create index on MongoDB. It's strange I don't understand mongodb what is is checking :thinking: 
```
test> db.Doubles.createIndex({v: 1}, {unique: true})
MongoServerError: Index build failed: 3c540fe6-c241-4c3f-a0e7-c0dc2c0af62a: Collection test.Doubles ( 6fceefe4-8aab-457a-afc5-f3696e9585b4 ) :: caused by :: E11000 duplicate key error collection: test.Doubles index: v_1 dup key: { v: -9.223372036854776e+18 }

```

Reply:
We have a test for creating a unique index for an existing field. I'm not sure that it depends on the data in a collection. I can't come up with a test case that would trigger a unique violation in PG.  I created an issue for adding this test case.

Reply:
This one is just totally strange, but yeah let's not make it part of this issue, it looks I don't understand how mongodb works :see_no_evil: